extension/oteleventually/eventstore.go

```diff
@@ -159,7 +158,10 @@ func (es *InstrumentedEventStore) Append(
 		}
 	}()
 
-	ctx, span := es.tracer.Start(ctx, "EventStore.Append", trace.WithAttributes(attributes...))
+	spanAttributes := attributes
```

Comment:
`attributes` is a slice, i.e. a reference. shouldn't this be something like
```go
spanAttributes := make([]attribute.KeyValue, len(attributes) + 1) // +1 for the extra stream name
copy(spanAttributes, attributes)
```
?

Reply:
Apparently, this is not necessary: https://go.dev/play/p/nmXf8GXs1yz

It might be due to the compiler recognizing the assignment of `append()` to a different variable 🤷🏻‍♂️

Reply:
I did it originally like so, but the linter was complaining about assigning the result of append to a different variable.

Reply:
Btw you can silence the linter, like I did here:
https://github.com/get-eventually/go-eventually/blob/75e48fd2b585a33dd2cdac7fcad82c1f113c269d/extension/oteleventually/projection.go#L83

Reply:
> It might be due to the compiler recognizing the assignment of append() to a different variable 🤷🏻‍♂️

You're right, Go is doing something extra again. Even if capacity of underlying array is enough for adding a new element, it "reslices" the array: https://pkg.go.dev/builtin#append TIL